roles/lib_openshift/library/oc_serviceaccount.py

```diff
@@ -1308,7 +1308,7 @@ def __init__(self, sname, namespace, kubeconfig, secrets=None, image_pull_secret
         self.create_dict()
 
     def create_dict(self):
-        ''' return a properly structured volume '''
+        ''' instantiate a properly structured volume '''
```

Comment:
hmm, it setups up all the parameters of what the object should look like.  Its sort of an instantiation